tests/components/backup/test_store.py

```diff
@@ -200,6 +200,49 @@ def mock_delay_save() -> Generator[None]:
             "minor_version": 4,
             "version": 1,
         },
+        {
+            "data": {
+                "backups": [
+                    {
+                        "backup_id": "abc123",
+                        "failed_agent_ids": ["test.remote"],
+                    }
+                ],
+                "config": {
+                    "agents": {
+                        "test.remote": {
+                            "protected": True,
+                            "retention": {"copies": None, "days": None},
+                        }
+                    },
+                    "automatic_backups_configured": True,
+                    "create_backup": {
+                        "agent_ids": [],
+                        "include_addons": None,
+                        "include_all_addons": False,
+                        "include_database": True,
+                        "include_folders": None,
+                        "name": None,
+                        "password": "hunter2",
+                    },
+                    "last_attempted_automatic_backup": None,
+                    "last_completed_automatic_backup": None,
+                    "retention": {
+                        "copies": None,
+                        "days": None,
+                    },
+                    "schedule": {
+                        "days": [],
+                        "recurrence": "never",
+                        "state": "never",
+                        "time": None,
+                    },
+                },
+            },
+            "key": DOMAIN,
+            "minor_version": 6,
```

Comment:
Why do we test migrating from minor version 6?

Reply:
I think I added it as a test that shouldn't break if we bump minor version in the future.